### PR TITLE
Log/error message improvement/fix in the xml coverage generator

### PR DESCRIPTION
**Pull request by krocard**
Improve and fix various error/log messages of the python report generator python script.

### CODE REVIEW

File: tools/coverage/coverage.py [hunk 1]
```diff
@@ -834,7 +834,7 @@ def _digest(self, lineLogType, lineLog):
 
 
     def parsePFWlog(self, lines):
-        for lineNb, lineLog in enumerate(lines):
+        for lineIndex, lineLog in enumerate(lines):
```
Comment (dawagner) [inline tools/coverage/coverage.py]: The offset may be given here:

    enumerate(lines, 1)
    # or
    enumerate(lines, start=1)

see https://docs.python.org/3/library/functions.html#enumerate

**Comment by dawagner** [comment]
We should add some unit tests for this coverage script, one of these days (same for the edd parser and xml generator and espacially those, since we use them more often).

File: tools/coverage/coverage.py (hunk 1 above)
Comment (krocard) [inline tools/coverage/coverage.py]: Good point

**Comment by krocard** [comment]
@dawagner A first easy step would be to generate an output given an input and compare with a reference. As those scripts are stateless, it should cover most of the code.

**Pull request merged**